spec/factories/tip.rb

```diff
@@ -16,7 +16,7 @@
   factory :checkin_tip, class: Tip do
     target 0
     partial_name 'checkin'
-    title 'エピソードにチェックインしよう'
+    title 'エピソードに記録しよう'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.